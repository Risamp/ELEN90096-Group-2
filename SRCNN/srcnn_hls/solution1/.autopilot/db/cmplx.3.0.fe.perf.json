{
  "0": {
    "basicblocks": 21,
    "calls": "1",
    "file": "srcnn.cpp",
    "fileloc": 4,
    "instructions": 95,
    "loads": {
      "G": 3,
      "T": 5
    },
    "name": "srcnn",
    "stores": {
      "G": 2,
      "T": 1
    },
    "total_instructions": 283
  },
  "1": {
    "basicblocks": 38,
    "calls": "",
    "file": "conv1.cpp",
    "fileloc": 9,
    "instructions": 188,
    "loads": {
      "G": 3,
      "L": 9
    },
    "name": "conv1",
    "stores": {
      "G": 6
    },
    "total_instructions": 188
  }
}