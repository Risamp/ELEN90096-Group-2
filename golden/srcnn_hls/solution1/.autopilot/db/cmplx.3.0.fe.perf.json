{
  "0": {
    "basicblocks": 33,
    "calls": "",
    "file": "srcnn.cpp",
    "fileloc": 4,
    "instructions": 153,
    "loads": {
      "G": 4,
      "T": 8
    },
    "name": "srcnn",
    "stores": {
      "G": 4,
      "T": 1
    },
    "total_instructions": 153
  }
}